addons/survey/views/survey_templates_user_input_session.xml

```diff
@@ -107,14 +107,14 @@
                     </div>
                 </div>
             </div>
-            <div class="o_container_small px-4 pb-3 pt96 d-flex flex-column o_survey_session_manage_container">
-                <div class="o_survey_session_description_done d-none">
+            <div class="px-sm-4 pb-3 pt96 d-flex flex-row align-items-center w-100 o_survey_session_manage_container">
```

Comment:
Hello @aath-odoo :smile: 
Thanks for updating! 

I think you can also remove this change:
`flex-row align-items-center w-100` 
now that we're using the `max-width` trick :slightly_smiling_face: